acknack_test.go

```diff
@@ -105,6 +108,35 @@ func (r *Runner) aCDSStreamWasInitiatedWithADiscoveryRequestMatchingYaml(arg1 *g
 	return nil
 }
 
+func (r *Runner) iSubscribeToDeltaCDSForTheseClusters(clustersYaml *godog.DocString) error {
```

Comment:
This function is currently not working, as the first response when doing stream.Recv() is an EOF error, indicating to me that there were no responses set. I am likely not setting this up correctly, but cannot tell at the moment where my understanding is off.